base/compiler/ssair/slot2ssa.jl

```diff
@@ -1,13 +1,5 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-struct TypedSlot
```

Comment:
Worth mentioning that deleting this definition breaks JuliaInterpreter (and therefore Revise)

Reply:
We'll put in a version check.